usr.sbin/pkg64c/Makefile

```diff
@@ -0,0 +1,4 @@
+PROG=	pkg64c
```

Comment:
Not sure I'd bother with this, just:
```
.if ${MACHINE_ABI:Mpurecap}
PROG?= pkg64c
.else
.PROG?= pkg${PKG_SUFFIX}
.endif
```
or whatever in usr.sbin/pkg/Makefile? Otherwise won't it build a purecap pkg twice and just clobber the final pkg binary for the pkg build with pkg.sh?

Reply:
(Yes, yes it will, it'll build and install pkg as part of usr.sbin/pkg then a later step will install pkg.sh over the top)

Reply:
Isn't usr.sbin/pkg no longer recursed into as that line is now removed from the Makefile?  I do think though that it might be simplest to do as you suggest though and avoid changing usr.sbin/Makefile entirely.  It's really just using an alternate name for the binary on purecap worlds vs installing a binary under a non-default ABI.

Reply:
If we want to install `pkg` for a hybrid world, it would be better to place this logic in `usr.sbin/Makefile` so that it's better visible, e.g.

`usr.sbin/Makefile`:
```
.if ${MACHINE_ABI:Mpurecap}
# Install pkg64 and pkg64c with pkg.sh instead of pkg for a purecap world.
SUBDIR.${MK_PKGBOOTSTRAP}.${MK_LIB64}+= pkg64
SUBDIR.${MK_PKGBOOTSTRAP}+=     pkg64c
.else
SUBDIR.${MK_PKGBOOTSTRAP}+=     pkg
.endif
```

`usr.sbin/pkg/Makefile`:
```
.if (${MACHINE_ABI:Mpurecap} && empty(PKG_SUFFIX))
# Install pkg.sh as part of pkg64c if world is purecap.
SCRIPTS=        pkg.sh
.endif
```

Reply:
> (Yes, yes it will, it'll build and install pkg as part of usr.sbin/pkg then a later step will install pkg.sh over the top)

As @bsdjhb said, the currently reviewed version has `usr.sbin/pkg` removed from `usr.sbin/Makefile`.

Reply:
I added this logic in `usr.sbin/Makefile`.